evm_loader/program/src/account_data.rs

```diff
@@ -6,6 +6,11 @@ use solana_program::{
     pubkey::Pubkey,
 };
 
+/// Ethereum account version
+pub const ACCOUNT_VERSION: u32 = 1_u32;
```

Comment:
I suggest renaming to `ACCOUNT_SEED_VERSION` and change the type to u8. In the feature, we can increase it to u16 without any problems.